ansible/roles/test/tasks/copp.yml

```diff
@@ -37,6 +37,7 @@
         ptf_qlen: 100000
         ptf_test_params:
         - verbose=False
+        - pkt_tx_count={{ pkt_tx_count|default(None) }}
```

Comment:
I'd better used something like:
default(0)
and in the test I'd use 
if self.pkt_tx_count == 0:
  self.pkt_tx_count = PKT_TX_COUNT

I don't know better way to omit pkt_tx_count from ptf_test_params in Ansible.
Probably 
{% if pkt_tx_count is defined %} - pkt_tx_count={{ pkt_tx_count }} {% endif %}
will work but I didn't check it